lab_1/start.py

```diff
@@ -19,7 +21,16 @@
             file_to_read:
         unknown_text = file_to_read.read()
 
+    en_profile = create_language_profile('en', en_text, ['a', 'the', 'is'])
+    print(en_profile)
+    de_profile = create_language_profile('de', de_text, ['und', 'ich', 'du'])
+    print(de_profile)
+    unknown_profile = create_language_profile('unknown', unknown_text, [])
+    print(unknown_profile)
+
     EXPECTED = 'en'
     RESULT = ''
+    RESULT = detect_language(unknown_profile, en_profile, de_profile, 3)
```

Comment:
can you explain in writing what happens in start.py?

Reply:
Here two functions are imported (create_language_profile and detect_language), the result of the work is shown (if functions work, then the whole program works) and at the end language is detected using language profiles. In create_language_profile a profile for each language is created and the success of the function is shown (the function works, so all functions that are called in create_language_profile work). In detect_language, the function for comparing two language profiles is called (two language profiles are compared and the name of the unknown language is output)